app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html

```diff
@@ -1,25 +1,18 @@
 @(params: Map[String, Any])
-@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Fulfilment House Due Diligence Scheme: Application received"){
-<p style="margin: 0 0 30px; font-size: 19px;">
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Fulfilment House Due Diligence Scheme application received"){
 
 <p style="margin: 0 0 30px; font-size: 19px;">Dear Customer</p>
-
 <p style="margin: 0 0 30px; font-size: 19px;">We have received your application to register a business for the Fulfilment House Due Diligence Scheme.</p>
-
-<p style="margin: 0 0 30px; font-size: 19px;"><h2>What happens next</h2></p>
+<p style="margin: 0 0 30px; font-size: 19px; font-weight: bold;">What happens next</p>
 <p style="margin: 0 0 30px; font-size: 19px;">We will review your application. We may get in touch with your contact person to discuss your application.</p>
 <p style="margin: 0 0 30px; font-size: 19px;">We will let you know when a decision has been made.</p>
-
-<p style="margin: 0 0 30px; font-size: 19px;"><h2>Help using this service</h2></p>
-<p style="margin: 0 0 30px; font-size: 19px;">See the guidance on GOV.UK to find out:
-<ul>
+<p style="margin: 0 0 30px; font-size: 19px; font-weight: bold;">Help using this service</p>
+<p style="margin: 0 0 30px; font-size: 19px;">See the guidance on GOV.UK to find out:</p>
+<ul style="margin: 0 0 30px; font-size: 19px;">
     <li>what you need to do while HMRC processes your application</li>
-    <li>how to make changes to your application</li>
     <li>how to contact HMRC about your application</li>
 </ul>
-</p>
-
 <p style="margin: 0 0 30px; font-size: 19px;">From the HMRC Fulfilment House Due Diligence Scheme</p>
+<p style="margin: 0 0 30px; font-size: 19px;">Do not reply to this email.</p>
```

Comment:
not needed as its already in the footer

Reply:
ah thanks - I saw that but noticed the don't reply instruction was only if user's were not sure if email was really from HMRC or not. We can take it out.